dev/conductor/ui/pubspec.yaml

```diff
@@ -14,18 +14,28 @@ dependencies:
     sdk: flutter
   platform: 3.0.2
 
+  archive: 3.1.4 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```

Comment:
this is a new package and this is the first pub roll since it was introduced--all of the added dependencies are transitive ones that were already implicitly included before.